### DISCUSSION TITLE
✨ Add Mention Feature for Task Assignment in TinaCMS

### DISCUSSION THREAD

**Discussion by ssw-yakshaver** [tinacms/tinacms#5250] [Ideas]
cc: @wicksipedia, @wicksipedia, @adamcogan, @JackDevAU

Hi Team!

[Watch the video]()

### Pain
As a user, I want to be able to mention other users when creating a new task in TinaCMS, so that they are notified and can take appropriate action.

### Acceptance Criteria
1. User can type '@' followed by a username to mention another user in the task description.
2. Mentioned users receive a notification about the task.
3. The mention feature should work seamlessly with Markdown, MDX, and JSON content formats.
4. The feature should be compatible with both self-hosted and TinaCloud-hosted instances of TinaCMS.

Requested by: @tomek-i

**Comment by tomek-i** [comment]
testing a potential bug, can be closed.

https://github.com/orgs/SSWConsulting/projects/59/views/1?pane=issue&itemId=83370485&issue=SSWConsulting%7CSSW.YakShaver%7C861


seems to be resolved.
